uiowa/uiowa
# [ ITS ] fix funky config

closes #8272
closes #8241

## To test
1. Sync down the site
2. Export conig and observe that it does not error out and fail. It may warn you, but that doesn't count!
3. check some service pages to see that they now have id's that allow you to make anchors to them
    i. (eg. #alerts, #support-training, #news)
